package.json

```
@@ -85,7 +86,8 @@
         "docker:test": "docker compose -f compose.yml -f docker-compose/compose.test.yml --profile tests up -d --build && docker compose -f compose.yml -f docker-compose/compose.test.yml attach cypress-test",
         "docker:stop": "docker compose -f compose.yml -f docker-compose/compose.dev.yml -f docker-compose/compose.test.yml kill",
         "docker:down": "docker compose -f compose.yml -f docker-compose/compose.dev.yml -f docker-compose/compose.test.yml down",
-        "docker:prune": "docker compose -f compose.yml -f docker-compose/compose.dev.yml -f docker-compose/compose.test.yml down -v && docker network prune && docker network rm mnestix-network"
+        "docker:prune": "docker compose -f compose.yml -f docker-compose/compose.dev.yml -f docker-compose/compose.test.yml down -v && docker network prune && docker network rm mnestix-network",
+        "docker:keycloak": "docker compose -f compose.yml -f docker-compose/compose.keycloak.yml up -d"
```

I think it makes sense to run it in dev mode here, as the keycloak stuff will probably mostly be used in development anyways
```suggestion
        "docker:keycloak": "docker compose -f compose.yml -f docker-compose/compose.dev.yml -f docker-compose/compose.keycloak.yml up -d"
```